### PR TITLE
[14.0][FIX] sign_oca: Some fixes

### PR DESCRIPTION
**Pull request by PauBForgeFlow**
1. Add _Sign_ button in request form (it was only in tree view).
2. Fix configure template action when it is created from a request.

### CODE REVIEW

**Review by BernatPForgeFlow — APPROVED** [review]

File: sign_oca/models/sign_oca_request.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ class SignOcaRequest(models.Model):
     def _compute_next_item_id(self):
         for record in self:
             record.next_item_id = (
-                record.signatory_data and max(record.signatory_data.keys()) or 0
+                record.signatory_data and int(max(record.signatory_data.keys())) or 0
```
Comment (etobella) [inline sign_oca/models/sign_oca_request.py]: Reviewing this is probably a bad decision, we should need to do:
`max([int(key) for key in record.signatory_data.keys()])`

For example, if you do max(['9', '10']) the result is '9'

File: sign_oca/models/sign_oca_request.py [hunk 2]
```diff
@@ -102,7 +102,7 @@ def _ensure_draft(self):
 
     def configure(self):
         self._ensure_draft()
-        self._set_action_log("configure")
+        self._set_action_log("view")
```
Comment (etobella) [inline sign_oca/models/sign_oca_request.py]: Why do you change this?

**Review by etobella — CHANGES_REQUESTED** [review]

File: sign_oca/models/sign_oca_request.py (hunk 2 above)
Comment (PauBForgeFlow) [reply sign_oca/models/sign_oca_request.py]: There is not an action named "configure". Another change could be to create it.

Comment (victoralmau) [reply sign_oca/models/sign_oca_request.py]: IMO the correction should be https://github.com/OCA/sign/pull/8/commits/e0591479ae850c5b7b02cb7d563fb5f49d2ff2c3

Comment (etobella) [reply sign_oca/models/sign_oca_request.py]: I do agree with @victoralmau

Comment (PauBForgeFlow) [reply sign_oca/models/sign_oca_request.py]: Deleted. This will be fixed with @victoralmau commit.

**Comment by pedrobaeza** [comment]
Can you rebase?

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Comment by pedrobaeza** [comment]
@etobella I have launched the merge command as your requests were attended.

**Pull request merged**